### PR TITLE
feat: describe Matrix move

### PR DESCRIPTION
**Pull request by paddatrapper**
Fixes: https://github.com/libretime/organization/issues/33

### CODE REVIEW

File: blog/2023-05-09-moving-to-matrix.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   account already)
+2. Search for #libretime:matrix.org or open
+   https://matrix.to/#/#libretime:matrix.org.
+3. Join the Space and select the rooms you wish to join
+
+## Accessing the past Mattermost chat data
+
+TODO: Develop a method of accessing the chat log for the various rooms is
+accessible by the users in Mattermost without making it public.
+Password-protected?
```
Comment (jooola) [inline blog/2023-05-09-moving-to-matrix.md:42]: I think that's too much work, for not much added value.

What about sharing the data on user request, through an email request  (maybe don't print the email in the blog post, I hate spam :smile:) ?

Comment (paddatrapper) [reply blog/2023-05-09-moving-to-matrix.md:42]: That would work I think

**Comment by jooola** [comment]
Looks good to me, do you have a timeline of what should happen when ?

**Comment by jooola** [comment]
Alright, feel free to merge this.

**Pull request merged**